config/initializers/dor_config.rb

```diff
@@ -33,12 +33,6 @@
     timeout Settings.workflow.timeout
   end
 
-  dor_services do
```

Comment:
With this (as-of-this-PR unused) configuration in place, both Dor::Config and the dor-services-client initializer attempt to configure the dor-services-client, and that's bad because the former has no awareness of token-based auth, which results in lots of 401s in the test suite.